activities/github-doc-workflow.md

```diff
@@ -0,0 +1,98 @@
+# GitHub workflow for pull requests
+
+For some changes, it can be simpler and quicker to submit a pull request using
+only GitHub rather than using git and your local development
+environment. In this section, we'll edit the documentation for a repository
+that you do not have cloned locally (GMT) on GitHub.
```

Comment:
Just adding some links.

```suggestion
only [GitHub](https://github.com/) rather than using git and your local development
environment. In this section, we'll edit the documentation for a repository
that you do not have cloned locally ([GMT](https://github.com/GenericMappingTools/gmt)) on GitHub.
```